src/Microsoft.DotNet.Web.Spa.ProjectTemplates/Angular-CSharp.csproj.in

```diff
@@ -13,8 +13,13 @@
     <BuildServerSideRenderer>false</BuildServerSideRenderer>
     <!-- aspnet/BuildTools#662 Don't police what version of NetCoreApp we use -->
     <NETCoreAppMaximumVersion>99.9</NETCoreAppMaximumVersion>
+    <RuntimeFrameworkVersion>${MicrosoftNETCoreApp30PackageVersion}</RuntimeFrameworkVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <DotNetCoreRuntime Include="${MicrosoftNETCoreApp30PackageVersion}" />
```

Comment:
Remove this. This is a KoreBuild-ism, not a built-in SDK feature. This shouldn't be in the templates we give to users.